docs/modules/ROOT/pages/references/architecture/index.adoc

```diff
@@ -0,0 +1,34 @@
+= APPUiO Managed OpenShift 4 Architecture
+
+Every APPUiO Managed OpenShift 4 has 3 control plane nodes.
+Additionally, APPUiO Managed OpenShift 4 clusters have https://docs.openshift.com/container-platform/4.13/nodes/nodes/nodes-nodes-creating-infrastructure-nodes.html[4 infrastructure nodes].
+Finally, APPUiO Managed OpenShift 4 clusters have 3 worker nodes by default.
+The minimum size for each node is 4 vCPUs, 16 GiB of RAM and 100 GiB root disk.
```

Comment:
Omit adding specific values here. Instead, state that there are minimum requirements. In the infra specific architectures, we then should link to https://products.vshn.ch/appuio/managed/ocp4_cloudscale.html#_default_configuration_minimum_requirements.

Reply:
Adjusted to reflect the actual minimum requirements of Red Hat OpenShift 4. I looked through our variants, and they all have the same minimum requirements, so I left our base line configuration here, but I can remove it if that works better for you.